src/entities/EditorDocument/EditorDocument.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { EditorDocument } from './index';
+import { BlockNode, BlockNodeName, createBlockNodeName } from '../BlockNode';
+
+const createBlock = ({ name, parent }: { name?: BlockNodeName, parent: EditorDocument }): BlockNode => {
+  return new BlockNode({
+    name: name || createBlockNodeName('block'),
```

Comment:
```ts
createBlockNodeName('block')
```

naming is not clear for me. What's happening here?

Reply:
I'm creating a block node name here. Because we use Nominal type for this field. I can't just use 'string' here.

Reply:
so let's give more realistic name:

```ts
createBlockNodeName('header')
```

or at least 

```ts
createBlockNodeName('block-name-example')
```